web/src/cypress/support/form.ts

```diff
@@ -21,7 +118,15 @@ function fillFormField(
   return cy.get(selector).then(el => {
     const isSelect =
       el.parents('[data-cy=material-select]').data('cy') === 'material-select'
+    const pickerFallback = el
+      .parents('[data-cy-fallback-type]')
+      .data('cyFallbackType')
     if (isSelect) {
+      if (DateTime.isDateTime(value)) {
```

Comment:
nit: Can we add spacing to break all this apart a bit for readability? Not a big deal if you'd rather not but just a personal nitpick :)